omod-2.4/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_10/ProgramEnrollmentController1_10Test.java

```diff
@@ -182,8 +182,6 @@ public void shouldVoidPatientState() throws Exception {
 		
 		Assert.assertTrue(voidedPatientState.getVoided());
 		Assert.assertFalse(existingPatientState.getVoided());
-		Assert.assertNull(existingPatientState.getEndDate());
```

Comment:
Did you intentionally remove this?

Reply:
The uuid now points to COMPLETED patient state in standardTestDataset thus endDate is not nulled when the last state is voided. It's correct.